src/sdk/Exports.ts

```diff
@@ -50,3 +50,17 @@ export { ActivityReceivedEventArgs } from "./ActivityReceivedEventArgs";
 export { ServicePropertyChannel } from "./ServicePropertyChannel";
 export { ProfanityOption } from "./ProfanityOption";
 export { BaseAudioPlayer } from "./Audio/BaseAudioPlayer";
+export { Conversation,
+    ConversationExpirationEventArgs,
+    ConversationImpl,
+    ConversationParticipantsChangedEventArgs,
+    ConversationTranslationCanceledEventArgs,
+    ConversationTranslationEventArgs,
+    ConversationTranslationResult,
+    ConversationTranslator,
+    IConversation,
```

Comment:
>IConversation [](start = 4, length = 13)

Why expose both the interface and the abstract class?

Reply:
Do you mean for the Conversation / IConversation?
If I've got a Typescript client consuming the sdk, then I get a no exported member error if they aren't both exported, e.g.
const myConversation: sdk.IConversation = sdk.Conversation.createConversationAsync(myConfig);

Reply:
The interface is removed from the exports.ts